### DISCUSSION TITLE
Reminder does not move with appointment

### DISCUSSION THREAD

**Discussion by Bitfire-Development** [Bitfire-Development/snooze-it-feedback#69] [Issues]
_This issue was originally posted by 'Casmesterom' on uservoice and had 1 vote(s)_

Per your request - the issue happened again. Appointment moved; reminder on the original date/time

An appointment that was originally scheduled for today was moved to in my Samsung schedule to tomorrow - the reminder still went off today.
Also, if I now open your app the reminder -the one that got moved to tomorrow- is not showing as a scheduled reminder. I will know tomorrow if I will still get a reminder thru your app. I do hope so.
Other than this minor issue and the fact that for feedback one needs to  create an account while only trying to help,  this is fantastic!

**Comment by Bitfire-Development** [comment]
Hi, Thanks for reporting the issue.

Is there any chance you snoozed/moved the reminder before you moved the event?

When a reminder is imported from your calendar, it has a trigger that is linked to the event (for example: '5 Minutes before the event'). However, when you snooze or move that reminder, its trigger is often unlinked from the event (for example: 'Today, 12:30'). You can check the trigger by tapping on a scheduled reminder, and looking at the piece of text under the title.

If you want to move a reminder and you want it to stay linked to the event, you can select the 'Before/after event' option when moving a reminder (or when tapping 'Snooze Until...').

This problem could also occur when the calendar integration has not yet synced (in this case the app doesn't yet know the event moved). If this is the problem, you could fix it by lowering the synchronization interval of the calendar integration. You can find this setting under 'Integrations' -> 'System Calendar (Android)' -> 'Settings' (gear icon in top right corner) -> 'Synchronize every'.

If both of the above don't fix your issue, please let me know and I'll dig deeper into what might be causing this.

By the way, I'm sorry that you had to create an account. I wish UserVoice (this website) would let me allow users to leave feedback without an account.

**Reply by J99eow** [reply]
Hi Renzo,

I'm also having this issue. I don't know if it's been happening forever and I just now realized... or if it's something happening recently.

I've tried force syncing, but that doesn't make the appointment update in Snooze It! I've tried backing up, deleting the data in the Snooze It! app on my phone, restoring and re-syncing. No luck.

I'm on a Xiaomi 13T pro with Android 15. Feel free to ask me questions, give me instructions, and use me as a test case. I know Xiaomi phones are aggressive app killers for battery saving but I've got Snooze It! set to "No restrictions" in the UI - the least amount of battery management the phone offers.  

Thank you,

John
